csharp-architectural-patterns/ModularMonolithPatternInCSharp/Inventory/Inventory.csproj

```diff
@@ -0,0 +1,21 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="RabbitMQ.Client" Version="6.8.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Inventory.Models\Inventory.Models.csproj" />
+  </ItemGroup>
+
+	<ItemGroup>
```

Comment:
Indent wrong of the whole block, is that something we need to add? Never seen it to be honest :D

Reply:
We usually don't add this in a class library, but since this is a modular monolith, I distributed the controllers into each module to separate the concerns.
In a Web API project this is not necessary since it uses the `Microsoft.NET.Sdk.Web` which packs all the necessary assemblies. However, in a class library which uses the `Microsoft.NET.Sdk`, this is required.